src/puppetlabs/dujour/version_check.clj

```diff
@@ -106,7 +125,7 @@
   (let [update-server-url (or update-server-url default-update-server-url)
         {:keys [version newer link] :as response} (try
                                                     (update-info request-values update-server-url)
-                                                    (catch Throwable e
+                                                    (catch Exception e
                                                       (log/debug e (format "Could not retrieve update information (%s)" update-server-url))))
```

Comment:
Should this be `log/debug` or `log/error`? If it's debug I don't think users will ever see this (which is probably okay?) If it's error, they'll be able to see something like this if something goes wrong:

```
2016-08-31 10:18:54,947 ERROR [clojure-agent-send-off-pool-3] [p.d.version-check] Could not retrieve update information (http://localhost/)
java.net.ConnectException: Connection refused
       	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method) ~[na:1.8.0_25]
       	at sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:716) ~[na:1.8.0_25]
       	at org.apache.http.impl.nio.reactor.DefaultConnectingIOReactor.processEvent(DefaultConnectingIOReactor.java:173) ~[httpcore-nio-4.4.4.jar:4.4.4]
       	at org.apache.http.impl.nio.reactor.DefaultConnectingIOReactor.processEvents(DefaultConnectingIOReactor.java:147) ~[httpcore-nio-4.4.4.jar:4.4.4]
       	at org.apache.http.impl.nio.reactor.AbstractMultiworkerIOReactor.execute(AbstractMultiworkerIOReactor.java:350) ~[httpcore-nio-4.4.4.jar:4.4.4]
       	at org.apache.http.impl.nio.conn.PoolingNHttpClientConnectionManager.execute(PoolingNHttpClientConnectionManager.java:191) ~[httpasyncclient-4.1.1.jar:4.1.1]
       	at org.apache.http.impl.nio.client.CloseableHttpAsyncClientBase$1.run(CloseableHttpAsyncClientBase.java:64) ~[httpasyncclient-4.1.1.jar:4.1.1]
       	at java.lang.Thread.run(Thread.java:745) [na:1.8.0_25]
```

It might be fine to hide that from them though since it doesn't really stop puppetdb/puppetserver/etc from working. Thoughts?